device-types/Extreme Networks/X460-G2-48x-10GE4.yaml

```diff
@@ -0,0 +1,186 @@
+    mgmt_only: false
+  - name: '13'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '14'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '15'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '16'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '17'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '18'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '19'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '20'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '21'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '22'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '23'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '24'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '25'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '26'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '27'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '28'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '29'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '30'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '31'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '32'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '33'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '34'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '35'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '36'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '37'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '38'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '39'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '40'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '41'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '42'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '43'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '44'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '45'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '46'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '47'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '48'
+    type: 1000base-x-sfp
+    mgmt_only: false
+  - name: '49'
+    type: 10gbase-x-sfpp
+    mgmt_only: false
+  - name: '50'
+    type: 10gbase-x-sfpp
+    mgmt_only: false
+  - name: '51'
+    type: 10gbase-x-sfpp
+    mgmt_only: false
+  - name: '52'
+    type: 10gbase-x-sfpp
+    mgmt_only: false
+  - name: '53'
+    type: other
+    mgmt_only: false
+    description: either Stacking-Ports or additional SFP+; only with VIM-2 module
+  - name: '54'
+    type: other
+    mgmt_only: false
+    description: either Stacking-Ports or additional SFP+; only with VIM-2 module
+  - name: Mgmt
+    type: 1000base-t
+    mgmt_only: true
+device-bays:
+  - name: PSU-1
+  - name: PSU-2
+  - name: VIM-2
```

Comment:
Is this VIM-2 an actual device, or is it a module?

Reply:
VIM-2 is a module with diferent types of ports
 - X460-G2 VIM-2x: 2 x 10gbase-x-sfpp
- X460-G2 VIM-2t: 2 x 10gbase-T
- X460-G2 VIM-2ss: 2 x extreme-summitstack
- X460-G2 VIM-2q: 2 x 40gbase-x-qsfpp

Reply:
In fact, one of this VIM-2 already exists in `device-types/Extreme Networks/X460-G2_VIM-2x.yaml`. But it has an issue I don't know of to solve it. Interface names are hardcoded to '33' and '34'; This is fine for `device-types/Extreme Networks/X460-G2-24t-10GE4.yaml` but not for `device-types/Extreme Networks/X460-G2-48x-10GE4.yaml` where these ports are '53' and '54'. There's a way to solve this?'

Reply:
Yes, the correct way to solve this is by changing the interface name on the module type to `{module}`, and then on the device your module-bay should have the `position` element set to whichever interface name it should have

Reply:
Changed, but still got an issue with the numbering. The VIM module provides 2 ports. I can use `position`but what is really needed is `position` and `position+1`. Is there any way to describe this?